src/styles/index.scss

```diff
@@ -1,3 +1,75 @@
+$daySize: 100px;
+$dayGap: 1px;
+$containerPadding: 10px;
+$daysInMonths: 31;
+$daysOfWeek: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+* {
```

Comment:
Don't use * selector (it impacts performance), better use ``html`` or ``body`` tag.

![image](https://github.com/mate-academy/layout_calendar/assets/54860641/f1957358-d441-4475-ae6a-1f41dccb07a4)